cardigan_beta/stories/components/Promos.js

```diff
@@ -0,0 +1,196 @@
+import { storiesOf } from '@storybook/react';
+import { doc, withReadme }  from 'storybook-readme';
+import { text, boolean, select } from '@storybook/addon-knobs/react';
+import { id, url, image, imageTall } from '../content';
+import moment from 'moment';
+import PromoReadme from '../../../common/views/components/Promo/README.md';
+import EditorialPromoReadme from '../../../common/views/components/Promo/README-editorial.md';
+import Promo from '../../../common/views/components/Promo/Promo';
+import EventPromoReadme from '../../../common/views/components/EventPromo/README.md';
+import EventPromo from '../../../common/views/components/EventPromo/EventPromo';
+import ExhibitionPromoReadme from '../../../common/views/components/ExhibitionPromo/README.md';
+import ExhibitionPromo from '../../../common/views/components/ExhibitionPromo/ExhibitionPromo';
+import WorkPromoReadme from '../../../common/views/components/WorkPromo/README.md';
+import WorkPromo from '../../../common/views/components/WorkPromo/WorkPromo';
+
+const sizes = '(min-width: 1340px) calc(15vw + 120px), (min-width: 960px) calc(40vw - 84px), (min-width: 600px) calc(60vw - 83px), calc(75vw - 72px)';
+const datePublished = '1685';
+
+const EditorialPromoExample = () => {
+  const title = text('Title', 'Some sort of title');
+  const description = text('Description', 'A description goes here a description goes here');
+  const contentType = select('Content type', [
+    'article',
+    'comic',
+    'audio',
+    'video',
+    'gallery',
+    'place',
+    null
+  ], 'article');
+  const series = [{'name': 'Searching for Genius', 'description': [{'type': 'paragraph', 'text': 'This five-part series is great. ', 'spans': []}], 'color': 'turquoise', 'commissionedLength': 6, 'schedule': [{'title': [{'type': 'heading1', 'text': 'First heading', 'spans': []}], 'publishDate': '2018-04-18T23: 00: 00+0000'}, {'title': [{'type': 'heading1', 'text': 'Second heading', 'spans': []}], 'publishDate': '2018-04-25T23: 00: 00+0000'}, {'title': [{'type': 'heading1', 'text': 'Third heading', 'spans': []}], 'publishDate': '2018-05-02T23: 00: 00+0000'}, {'title': [{'type': 'heading1', 'text': 'Fourth heading', 'spans': []}], 'publishDate': '2018-05-09T23: 00: 00+0000'}, {'title': [{'type': 'heading1', 'text': 'Fifth heading', 'spans': []}], 'publishDate': '2018-05-16T23: 00: 00+0000'}], 'promo': [], 'wordpressSlug': null}];
```

Comment:
Are we likely to need a series for other stories, and if so would it make sense to move this to a e.g. `mocks` directory?

I guess we could do that at the point it becomes necessary.

Reply:
That's what the `content.js` file is for?
Happy to call it mocks.

Reply:
Ah cool, `mocks` was just an example – if it already exists as `content` I'm happy with that.